src/App.cpp

```diff
@@ -40,7 +40,7 @@ void App::Update() {
     }
 }
 
-void App::End() {
+void App::End() const {
     Core::Context::GetInstance()->SetExit(true);
```

Comment:
`SetExit` is unnecessary here. And `App::End` is meant to be modifiable by developers and should not be const.